docs/manuals/auth/index.md

```diff
@@ -0,0 +1,53 @@
+# Taipy's Authentication and Authorization
+
+!!! warning "Reminder: Authorization and Authentication is only available in enterprise version"
+
+## Pre-requisite
+
+An LDAP server must be setup by the user. Taipy will not be responsible for setting the server up.
+
+## Config
+
+Set up connection to your LDAP server with Taipy Config. Here are the configurable properties you can use:
+
+- **Config.global_config.ldap_server**: The url of the ldap server.
+- **Config.global_config.ldap_base_dn**: The base DN that the LDAP server uses to search for users authentication.
+- **Config.global_config.jwt_secret**: The secret for the JWT token.
+- **Config.global_config.jwt_expire**: The period after which the JWT token provided by Taipy will be expired.
+- **Config.global_config.auth_enabled**: The field to enable or disable Taipy authentication and authorization. Its default value is `True`, meaning that the enterprise version, by default, will have authentication and authorization feature enabled.
+
+Example of configuring LDAP server connection in Taipy:
+
+```
+Config.global_config.ldap_server = "ldap://0.0.0.0"
+Config.global_config.ldap_base_dn = "dc=example,dc=org"
+Config.global_config.jwt_secret = "my-32-character-ultra-secure-and-ultra-long-secret"
+Config.global_config.jwt_expire = 600   #for 10 minutes
+Config.global_config.auth_enabled = True
+```
```

Comment:
```suggestion

=== "Python configuration"

    ```
    Config.configure_global_app(ldap_server="ldap://0.0.0.0", 
                                ldap_base_dn="dc=example,dc=org",
                                jwt_secret = "my-ultra-secure-and-ultra-long-secret",
                                jwt_expire = 600   #for 10 minutes,
                                auth_enabled = True)
    ```

=== "TOML configuration"

    ```python linenums="1"
    from taipy import Config

    Config.load("config.toml")
    ```

    ```toml linenums="1" title="config.toml"

    [TAIPY]

    ldap_server="ldap://0.0.0.0"
    ldap_base_dn="dc=example,dc=org"
    jwt_secret = "my-ultra-secure-and-ultra-long-secret"
    jwt_expire = 600   #for 10 minutes
    auth_enabled = True
    ```
```